docs/themes/hugo-docs/layouts/_default/article.html

```diff
@@ -3,16 +3,6 @@
 {{ $versions := split $versionString "," }}
 {{ $latestVersion := index $versions 0 }}
 
-{{ if (eq $currentVersion "master") }}
```

Comment:
Can we remove the currentVersion variable? I don't think we use it anywhere.

Reply:
I can remove `currentVersion`, but I should check if it's used anywhere else or not to avoid any issues

Reply:
I've removed the unused variable and checked that it doesn't create any issues